src/test/java/org/folio/rest/support/db/ErrorBuilder.java

```diff
@@ -0,0 +1,104 @@
+package org.folio.rest.support.db;
+
+import scala.Tuple2;
+import scala.collection.immutable.Map;
+import scala.collection.immutable.Map$;
+
+public class ErrorBuilder {
```

Comment:
What kind of error does this build?

Has this and the other general classes below been copied from another project?

Reply:
You are right. I have copied then from another project

Reply:
Which project was it copied from?

Reply:
folio-service-tools

Reply:
Why did you choose to copy this code instead of introduce a dependency on the folio-service-tools library?